Description: Adapt HTML serialization test for libxml 2.15
Author: Antonin Delpeuch <antonin@delpeuch.eu>
Bug: https://github.com/KWARC/rust-libxml/issues/187
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1119024
Forwarded: not-needed
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/tests/resources/as_html_result.xml
+++ b/tests/resources/as_html_result.xml
@@ -1,5 +1,5 @@
 <html>
-<head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+<head><meta charset="UTF-8">
 <title>Page Title</title>
 </head>
 <body>
